demos/python_demos/asl_recognition_demo/asl_recognition_demo/video_stream.py

```diff
@@ -0,0 +1,154 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import time
+import signal
+from os.path import exists
+from multiprocessing import Process, Array
+
+import cv2
+import numpy as np
+
+
+class VideoStream:
+    """ This class returns constant framerate frames from the input stream. """
+
+    def __init__(self, path, trg_fps, batch_size):
+        """Conctructor"""
+
+        if not exists(path):
+            raise Exception('File not found: {}'.format(path))
+
+        self._trg_fps = trg_fps
+        assert self._trg_fps > 0
+        self._batch_size = batch_size
+        assert self._batch_size > 0
+
+        self._cap = cv2.VideoCapture(path)
```

Comment:
Please convert the path to an `int` if it is an integer. That will make it possible to open cameras in a platform-independent way.

Reply:
Done.